Dockerfile

```diff
@@ -46,5 +46,6 @@ LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
 COPY --from=setup /home/carma/install /opt/carma/install
+RUN sudo git clone https://github.com/NewEagleRaptor/pacifica-dbw-ros.git /opt/pacifica-dbw-ros --branch master --depth 1 &&  sudo cp /opt/pacifica-dbw-ros/dbw_pacifica_can/New_Eagle_DBW_3.3.388.dbc /opt/carma/install/share/dbw_pacifica_can/New_Eagle_DBW_3.3.388.dbc
```

Comment:
at the time cp command there does not work because the install path does not exist after copying i can remove the source code or we can put that file in the configuration repo

Reply:
You can just copy it to a temporary location in the home folder and then copy it back out at the install step. Or if you want to copy from the clone source code then you should delete the clone repo afterward. I just want to preserve the fact that the final image does not contain source code

Reply:
I removed the cloned folder after copying the license file.